media/js/base/nav-main.js

```diff
@@ -14,7 +14,7 @@ $(document).ready(function() {
     var newItem;
     var focusedItem;
 
-    $('#nav-main > .has-submenus > li').bind('mouseover focusin', function() {
+    $('#nav-main > .has-submenus > li').on('mouseover focusin', function() {
```

Comment:
Is this file still used anywhere? I can't see it in any bundles

Reply:
Good catch. Removing!